### PR TITLE
plop malloy views in the main explorer, only show them if a .malloy f…

### PR DESCRIPTION
**Pull request by bporterfield**
…ile is opened

unclear if this is better. maybe?

### CODE REVIEW

**Comment by bporterfield** [comment]
https://user-images.githubusercontent.com/108260/182693408-5ff5f38b-471d-4abb-90a6-d2c6e686ecc1.mov

**Comment by christopherswenson** [comment]
Yeah this seems better to me? Definitely improves visibility at the expense of those views being organized into their own place

**Comment by anikaks** [comment]
I'm in favor! It won't mess with anyone's experience with using VS Code when they aren't in malloy files so it seems unlikely to be an annoyance and we can always undo if folks complain.

**Comment by whscullin** [comment]
Might be possible to have it stick around when a results tab is selected?

**Comment by bporterfield** [comment]
@whscullin that seems like a great idea - unfortunately I've so far been unsuccessful trying to get a when clause that will actually work for a generated webview. Still searching....

**Comment by bporterfield** [comment]
https://code.visualstudio.com/api/references/when-clause-contexts#inspect-context-keys-utility 

you...can't click on the webview and get any info :(

File: packages/malloy-vscode/package.json [hunk 1]
```diff
@@ -175,19 +175,22 @@
...
     ],
     "views": {
-      "malloy": [
+      "explorer": [
         {
           "id": "malloyHelp",
           "name": "Help",
-          "type": "webview"
+          "type": "webview",
+          "when": "resourceFilename =~ /malloy/"
```
Comment (whscullin) [inline packages/malloy-vscode/package.json:183]: I'd recommend `"resourceFilename =~ /\\.malloy$/"` to avoid false positives like `malloy.txt`

Comment (bporterfield) [reply packages/malloy-vscode/package.json:183]: 👍

**Comment by bporterfield** [comment]
@whscullin @christopherswenson now showing help/connections/schema when results panel is focused. Slightly annoying that there's a flash when clicking between .malloy file and results webview, but not that bad. LMK if y'all have any improvements.

**Pull request merged**